packages/typespec-client-generator-core/doc/types.tsp

```diff
@@ -77,7 +77,7 @@ model SdkBuiltInType extends SdkType {
     | "armId"
     | "ipAddress"
     | "azureLocation"
-    | "etag";
+    | "eTag";
```

Comment:
is that going to be breaking?

Reply:
kind of, but no one's using it rn. I'll make a note of it though